### PR TITLE
naming: create new vtctl commands 

### PR DESCRIPTION
**Pull request by deepthi**
## Description

Add `InitShardPrimary` and `SetShardIsPrimaryServing`
Deprecate `InitShardMaster` and `SetShardIsMasterServing`

## Related Issue(s)

#7114 

## Checklist
- [x] Tests were added or are not required
- [ ] Documentation was added or is not required

Note: Website docs will be updated after 11.0 release is published.

### CODE REVIEW

File: go/vt/wrangler/reparent.go [hunk 1]
```diff
@@ -125,9 +125,9 @@ func (wr *Wrangler) InitShardMaster(ctx context.Context, keyspace, shard string,
 		Force:                   force,
 	}, waitReplicasTimeout, wr.tmc, wr.logger)
 	if err != nil {
-		event.DispatchUpdate(ev, "failed InitShardMaster: "+err.Error())
+		event.DispatchUpdate(ev, "failed InitShardPrimary: "+err.Error())
 	} else {
-		event.DispatchUpdate(ev, "finished InitShardMaster")
+		event.DispatchUpdate(ev, "finished InitShardPrimary")
```
Comment (deepthi) [inline go/vt/wrangler/reparent.go:130]: What do you think of this? The event will be `InitShardPrimary` even if the command used by the user is `InitShardMaster`.

File: doc/SeparatingVttabletMysql.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ to make sure the tablet type in the topology is correct before running
 vttablet. Usually, vttablet can be started with `-init_tablet_type replica`,
 even for a master (as `master` is not allowed), and will figure out the master
```
Comment (ajm188) [inline doc/SeparatingVttabletMysql.md:34]: i know you are planning to do the website changes after 11.0, so i'm not sure if you want to tackle these at the same time but there's a bunch of "master" in this doc

File: go/test/endtoend/cluster/vtctlclient_process.go [hunk 3]
```diff
@@ -44,6 +44,9 @@ type VtctlClientParams struct {
 
 // InitShardPrimary executes vtctlclient command to make specified tablet the primary for the shard.
 func (vtctlclient *VtctlClientProcess) InitShardPrimary(Keyspace string, Shard string, Cell string, TabletUID int) (err error) {
+	// version_upgrade test depends on using older binaries
```
Comment (ajm188) [inline go/test/endtoend/cluster/vtctlclient_process.go:47]: 👍

File: go/vt/wrangler/reparent.go (hunk 1 above)
Comment (ajm188) [reply go/vt/wrangler/reparent.go:130]: Changing things now seems good to me. I don't see anything in the code or docs that indicates we rely on (or can expect to rely) the exact update message text.

File: go/vt/wrangler/schema.go [hunk 4]
```diff
@@ -333,7 +333,7 @@ func (wr *Wrangler) CopySchemaShardFromShard(ctx context.Context, tables, exclud
 		return fmt.Errorf("GetShard(%v, %v) failed: %v", sourceKeyspace, sourceShard, err)
 	}
 	if sourceShardInfo.MasterAlias == nil {
-		return fmt.Errorf("no master in shard record %v/%v. Consider running 'vtctl InitShardMaster' in case of a new shard or to reparent the shard to fix the topology data, or providing a non-master tablet alias", sourceKeyspace, sourceShard)
+		return fmt.Errorf("no master in shard record %v/%v. Consider running 'vtctl InitShardPrimary' in case of a new shard or to reparent the shard to fix the topology data, or providing a non-master tablet alias", sourceKeyspace, sourceShard)
```
Comment (ajm188) [inline go/vt/wrangler/schema.go:336]: should we change the other instances of "master" here (and the error below) to "primary"?

**Review by ajm188 — APPROVED** [review]
lgtm. A couple minor questions, nothing blocking!

File: go/vt/wrangler/schema.go (hunk 4 above)
Comment (deepthi) [reply go/vt/wrangler/schema.go:336]: Good catch, I'll fix them.

File: doc/SeparatingVttabletMysql.md (hunk 2 above)
Comment (deepthi) [reply doc/SeparatingVttabletMysql.md:34]: Once we get through all the known tasks I plan to do another `grep` through the code base and fix anything left behind.
I'll add this to the task list. So far I've been grepping only `go` files.

**Review by ajm188 — APPROVED** [review]

**Pull request merged**